spring-kafka-test/src/main/java/org/springframework/kafka/test/assertj/KafkaConditions.java

```diff
@@ -50,6 +51,15 @@ private KafkaConditions() {
 	}
 
 	/**
+	 * @param type the type of timestamp
+	 * @param value the timestamp.
+	 * @return a Condition that matches the timestamp value in a consumer record.
+	 */
+	public static Condition<ConsumerRecord<?, ?>> timestamp(TimestampType type, long value) {
```

Comment:
Looks like we are not using `KafkaMatchers` anywhere, may be safe to delete right?